doc/classes/Tween.xml

```diff
@@ -228,6 +228,13 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="set_ignore_time_scale">
+			<return type="Tween" />
+			<param index="0" name="ignore" type="bool" default="true" />
+			<description>
+				If [param ignore] is [code]true[/code], the speed of tweening will not be affected by [member Engine.time_scale]. This affects all [Tweener]s and their delays. Default value is [code]false[/code].
```

Comment:
Mixture between this PR's description and https://github.com/godotengine/godot/pull/100736 's <sub>which is taken from Timer and/or SceneTreeTimer, I believe, so it's consistent.</sub>
```suggestion
				If [param ignore] is [code]true[/code], the tween will ignore [member Engine.time_scale] and update with the real, elapsed time. This affects all [Tweener]s and their delays. Default value is [code]false[/code].
```